code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -5,13 +5,52 @@
 
 /obj/structure/closet/crate/necropolis
 	name = "necropolis chest"
-	desc = "It's watching you closely."
+	desc = "It's watching you suspiciously."
 	icon_state = "necrocrate"
 	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 	door_anim_time = 0
+	var/need_key = FALSE
```

Comment:
nitpick but I think `locked` would make more sense

Reply:
reworked the system to use `locked`